ts_src/psbt.ts

```diff
@@ -733,13 +739,12 @@ export class Psbt {
     });
 
     // generate output blinding factors
-    const numOutputs = this.data.outputs.length - 1; // exclude fees output
-    const outputAbfs = this.data.outputs
-      .slice(0, numOutputs)
-      .map(() => randomBytes(opts!));
-    const outputVbfs = this.data.outputs
-      .slice(0, numOutputs - 1)
-      .map(() => randomBytes(opts!));
+    const numOutputs = c.__TX.outs.length - 1 // excluding fee output
```

Comment:
this should not depend from outputs anymore.. it should be:
```suggestion
    const numOutputs = outputsIndexToBlind!.length
```